src/config/settings.js

```diff
@@ -1,35 +1,32 @@
-import { LOCAL_API_HOST } from './api';
+import { CONTEXTS } from './contexts';
+import { REACT_APP_MOCK_API } from './env';
 
-export const MAX_INPUT_LENGTH = 5000;
-export const MAX_ROWS = 10;
 export const DEFAULT_LANG = 'en';
-export const DEFAULT_MODE = 'student';
-export const TEACHER_MODE = 'teacher';
 
+const LOCAL_API_HOST = 'localhost:3000';
+
+// TODO: Delete.
```

Comment:
So shall we delete it?

Reply:
I have to test it in different context first.